src/hotspot/share/classfile/classLoader.cpp

```diff
@@ -285,9 +285,9 @@ ClassPathZipEntry::~ClassPathZipEntry() {
   FREE_C_HEAP_ARRAY(char, _zip_name);
 }
 
-u1* ClassPathZipEntry::open_entry(const char* name, jint* filesize, bool nul_terminate, TRAPS) {
+u1* ClassPathZipEntry::open_entry(Thread* current, const char* name, jint* filesize, bool nul_terminate) {
     // enable call to C land
-  JavaThread* thread = THREAD->as_Java_thread();
+  JavaThread* thread = current->as_Java_thread();
```

Comment:
No need to introduce a local for a single use.

Reply:
This is an existing issue but I've made the change and also fixed the indentation of the comment at line 289.